pom.xml

```diff
@@ -151,15 +152,31 @@
             <artifactId>jackson-datatype-jsr310</artifactId>
             <version>${jakson.datatype.version}</version>
         </dependency>
-
-
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-mail</artifactId>
+            <version>${email.version}</version>
```

Comment:
Do you really need version here? I believe it should be extended from parent

Reply:
done